src/prefect/tasks/airbyte/airbyte.py

```diff
@@ -76,7 +76,10 @@ def _check_health_status(self, session, airbyte_base_url):
         try:
             response = session.get(get_connection_url)
             self.logger.info(response.json())
-            health_status = response.json()["db"]
+            key = "db"
+            if "available" in response.json():
+                key = "available"
```

Comment:
Preference for the more "pythonic" variable assignment here

```suggestion
            key = "available" if "available" in response.json() else "db"
```